src/app.html

```diff
@@ -8,7 +8,8 @@
     <a href="#/tags">Tags</a>
     <a href="#/paths">Paths</a>
     <a href="#/types">Types</a>
-    <a href="javascript:void(0)" click.delegate="download('json')">Download</a>
+    <a href="javascript:void(0)" click.delegate="download('json')">Download (JSON)</a>
```

Comment:
Go ahead and nest these strings on their own line. That will improve readability, by letting us scan the text vertically and understand what is going on.